src/site/content/en/blog/coop-coep/index.md

```diff
@@ -357,121 +355,26 @@ isolated.
 
 ### Observe issues using the Reporting API
 
-The [Reporting
-API](https://developers.google.com/web/updates/2018/09/reportingapi) is another
-mechanism through which you can detect various issues. You can configure the
-Reporting API to instruct your users' browser to send a report whenever COEP
-blocks the loading of a resource or COOP isolates a popup window. Chrome has
-supported the
+The [Reporting API](/reporting-api) is another mechanism through which you can
+detect various issues. You can configure the Reporting API to instruct your
+users' browser to send a report whenever COEP blocks the loading of a resource
+or COOP isolates a pop-up window. Chrome has supported the
```

Comment:
How about making this sentence more generic?

"Chrome has supported the Reporting API since version 69 for a variety of uses including COEP and COOP."

Reply:
SGTM. Included your suggestion.